lib/daru/accessors/array_wrapper.rb

```diff
@@ -67,26 +67,24 @@ def dup
       end
 
       def mean
-        sum.quo(@size - @context.missing_positions.size).to_f
+        return nil if @data.compact.empty?
```

Comment:
`#only_valid` with `as_a: :array` returns an array without creating a vector. So, I can use `@context.only_valid(as_a: :array)` instead of `@data.compact`.